ClinGen/clincoded
# Connects to #718. Connects to #776. VCI Computation tab.

This PR consists of changes pertinent to the UI of computational tab.

**Known issues:**
1. The data in the "ClinGen Predictors" table is static at the moment. Will add logic to call the API (e.g. https://predictvar.bustamante-lab.net) after today.
2. Removed the placeholder external links in the "Repetitive Region" section for future implementation as they are low-priority. But the header of "Repetitive Region" is left in the code in case @mrmin123 needs it in the process of implementing the form. He can choose it to remove it or hide it together with the form.
3. Just discovered that not all ClinVar IDs (e.g. 139214) returns the <ProteinChange> node in its Eutils API response. Currently there is no logic to bypass it and thus rendering bails out due to error. Will add logic to fix this and the respective codon API call.

**Steps to test:**
1. Login and enter ClinVar ID 55847 on the variant selection page.
2. Navigate to the "Predictors" tab and expect to see real data being displayed in the "Other Predictors" and "Conservation Analysis" tables.
3. Click on various external links in the "Predictors" tab and expect to see the intended external pages.
